src/components/modal/TaskViewModal.tsx

```diff
@@ -12,6 +12,22 @@ interface ITaskViewModalProps {
   initialTask?: ToDo;
 }
 
+const stylesForModal = {
```

Comment:
Can't you create separate file for styles and import this object here?

Reply:
okay